src/Illuminate/Database/Connection.php

```diff
@@ -105,6 +105,13 @@ class Connection implements ConnectionInterface
      */
     protected $fetchMode = PDO::FETCH_OBJ;
 
+    /**
+     * The class to fetch as if $fetchMode is set to PDO::FETCH_CLASS.
+     *
+     * @var null|int
```

Comment:
This should be `string|null` right?

Reply:
Yes that is right. I'll change that, thanks!